Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: run
+run:
+	go run main.go
+
+# Run `make run-debug` in one terminal tab and `make print-log` in another to view the program and its log output side by side
+.PHONY: run-debug
+run-debug:
+	go run main.go -debug
+
+.PHONY: print-log
+print-log:
+	go run main.go --logs
+
+.PHONY: unit-test
+unit-test:
+	go test ./...
```

Comment:
We should probably add `-short` here to exclude the integration tests, since they have their own target. (Like we do [on CI](https://github.com/jesseduffield/lazygit/blob/master/.github/workflows/ci.yml#L46).)

Reply:
Thank you for pointing that out.
I fixed. 72dec599541f2b351860fdb4bff005166bb26480